### ISSUE TITLE
about code

### ISSUE THREAD

**Issue by zwzhang123** [JLDeng/ST-Norm#3]
x_list = []
x_list.append(x)
...
x = torch.cat(x_list, dim=1)

This part of the code is useless.

**Comment by Lucasxin** [comment]
This part of the code is useless？
